temporal-serviceclient/src/test/java/io/temporal/serviceclient/functional/GetResultsAsyncOverLongPollTimeoutTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.serviceclient.functional;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowStub;
+import io.temporal.serviceclient.functional.common.TestWorkflows;
+import io.temporal.testing.internal.SDKTestWorkflowRule;
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import java.util.concurrent.ExecutionException;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * These tests are intentionally running longer than a minute to cover a situation when getResult
+ * calls needs to survive a server timeout on a long poll of 60 seconds to get the result of
```

Comment:
Any reason not to just crank down the poll timeout a bit?

Reply:
I want to test a situation when the server responds with an empty result, not when we hit a long poll timeout (we ideally should never hit it). And for that, I just have to get that one minute. Or modify the settings of our server, which I don't want to abuse.